Server/JokesOnYou.Web.Api/DTOs/UserUpdateDTO.cs

```diff
@@ -7,5 +7,9 @@ namespace JokesOnYou.Web.Api.DTOs
 {
     public class UserUpdateDTO
     {
+        public string Name { get; set; } //Make sure that name is unique
+        public string Role { get; set; }
+        public bool Nsfw { get; set; } //Unsure if this means that the user posted nsfw or just that the user wants to see nsfw
```

Comment:
Yes we should clarify this. Nsfw means the user has agreed (true) to see nsfw jokes or hide them (false).
Maybe change it to SeeNsfwJokes ?
Not sure we would like a user to update his own role. and the name should require some checks but this works for now i guess ;)